packages/flutter/lib/src/widgets/default_text_editing_actions.dart

```diff
@@ -72,7 +72,7 @@ class _DoNothingAndStopPropagationTextAction extends TextEditingAction<DoNothing
   _DoNothingAndStopPropagationTextAction();
 
   @override
-  bool consumesKey(Intent intent) => true;
+  bool consumesKey(Intent intent) => false;
```

Comment:
Shouldn't this change the name to `_DoNothingTextAction`, since it no longer stops propagation?

Reply:
I thought the same thing, but look at the original DoNothingAndStopPropagation:

https://github.com/flutter/flutter/blob/f712fecde2e3105b6a2f604c86da86175444195d/packages/flutter/lib/src/widgets/app.dart#L1110

Is there a misnomer somewhere or are we misunderstanding what consumesKey means?

Reply:
My understanding is `consumesKey` has nothing to do with propagation. It's responsible for telling the engine whether the key event was handled or not.

Reply:
What stops the propagation is the fact that `DoNothingAndStopPropagationIntent` is, by default, attached to the `DoNothingAction`. But the `DoNothingAction` itself doesn't control propagation.